README.rst

```diff
@@ -72,10 +72,11 @@ in the source folder. Tests restricted to a specific provider can be run by exec
 
 .. note::
 
-    Tests on the `IBMQ device <https://pennylane-qiskit.readthedocs.io/en/latest/devices.html>`_ can
+    Tests on the `IBMQ device <https://pennylane-qiskit.readthedocs.io/en/latest/code/ibmq.html>`_ can
     only be run if a ``ibmqx_token`` for the
     `IBM Q experience <https://quantumexperience.ng.bluemix.net/qx/experience>`_ is
```

Comment:
Just noticed that this needs to be updated as well!
```suggestion
    `IBM Q experience <https://quantum-computing.ibm.com/>`_ is
```

Reply:
Amazing, thanks!